<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="1" parallel="classes" name="Failed suite [Arithmetic Function Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="1" parallel="classes" name="Arithmetic Function Unit Test(failed)">
    <groups>
      <run>
        <include name="SumTest"/>
        <include name="SubTest"/>
        <include name="MultiplyTest"/>
        <include name="DivTest"/>
      </run>
    </groups>
    <classes>
      <class name="org.rntgroup.CalcUnitTest.MultiplyDoubleTest">
        <methods>
          <include name="tearDown"/>
          <include name="SetUp"/>
          <include name="unitMultiplyDoubleTest" invocation-numbers="3 4"/>
        </methods>
      </class> <!-- org.rntgroup.CalcUnitTest.MultiplyDoubleTest -->
      <class name="org.rntgroup.CalcUnitTest.DivLongTest">
        <methods>
          <include name="tearDown"/>
          <include name="SetUp"/>
          <include name="unitDivByZeroLongTest"/>
        </methods>
      </class> <!-- org.rntgroup.CalcUnitTest.DivLongTest -->
    </classes>
  </test> <!-- Arithmetic Function Unit Test(failed) -->
</suite> <!-- Failed suite [Arithmetic Function Suite] -->
